android/app/src/main/java/org/pathcheck/covidsafepaths/exposurenotifications/storage/RealmSecureStorageBte.kt

```diff
@@ -95,17 +96,18 @@ object RealmSecureStorageBte {
         }
     }
 
-    fun refreshWithExposureWindows(exposureWindows: List<ExposureWindow>): Boolean {
+    fun refreshWithDailySummaries(dailySummaries: List<DailySummary>): Boolean {
```

Comment:
Is this storing the exposures that exceeded the threshold and led to notifications?
If so, please consider adding a function comment to specify that.

I initially thought this was ALL DailySummaries, and thought this wasn't needed anymore, since the API stores them for us.

Reply:
Yes, it only stores the date to avoid displaying the same notification twice.
I'll add a comment